setup.py

```diff
@@ -66,6 +66,11 @@
                "yt/utilities/lib/tsearch.c"],
               include_dirs=["yt/utilities/lib"],
               libraries=std_libs),
+    Extension("yt.geometry.ramses_oct_container",
```

Comment:
Maybe we should put this under the ramses frontend?  I don't know.

Reply:
This could actually be included in the octree_container as it is not Ramses specific anymore.